conans/client/generators/__init__.py

```diff
@@ -67,9 +67,9 @@ def __getitem__(self, key):
 registered_generators.add("virtualenv", VirtualEnvGenerator)
 registered_generators.add("virtualbuildenv", VirtualBuildEnvGenerator)
 registered_generators.add("virtualrunenv", VirtualRunEnvGenerator)
-registered_generators.add("boost-build", BoostBuildGenerator)
```

Comment:
I think we must not rename a generator because of the stability commitment of Conan 1.0. 
What do you think about creating a new generator "b2" and keep the old one deprecated? We could show a big warning in the old one (encouraging to use the new one) and remove it in Conan 2.0